src/main/java/frc/robot/commands/ShooterCommand.java

```diff
@@ -0,0 +1,46 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.commands;
+
+import edu.wpi.first.wpilibj2.command.Command;
+import frc.robot.subsystems.ShooterSubsystem;
+
+/** A Shooter Command that uses an example subsystem. */
+public class ShooterCommand extends Command {
+  @SuppressWarnings({"PMD.UnusedPrivateField", "PMD.SingularField"})
+  private final ShooterSubsystem m_ShooterSubsystem;
+
+  /**
+   * Creates a new ShooterCommand.
+   *
+   * @param subsystem The subsystem used by this command.
+   */
+  public ShooterCommand(ShooterSubsystem s_subsystem) {
+    m_ShooterSubsystem = s_subsystem;
```

Comment:
nit, can we rename to `shooterSubsystem`?

Reply:
done